File: 07Cadenas/numeric/a.cpp

#include <cmath>

int main(int argc, char const *argv[])
{
    double seno = 0, coseno = 0, tangente = 0;
    double logaritmo =0, raizCuad = 0;
    const double max = 10e7;
    for (double i = 1; i < max; ++i) {
        seno += sin(i);
        coseno += cos(i);
        tangente += tan(i);
        logaritmo += log(i);
        raizCuad += sqrt(i);
    }
    return 0;
}

File: 07Cadenas/numeric/a.py

import math 


seno = 0
coseno = 0 
tangente = 0;
raizCuad = 0;
logaritmo = 0
max_ = 10e7;
max_ = int(max_)
for i in range(1, max_):
    seno += math.sin(i);
    coseno += math.cos(i);
    tangente += math.tan(i);
    logaritmo += math.log(i);
    raizCuad += math.sqrt(i);